### PR TITLE
adding support for more notebooks in CI

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/_scripts/prepare_notebooks_for_ci.py [hunk 1]
```diff
@@ -16,6 +16,8 @@
     "WebBaseLoader",
     # skip if has draw_mermaid_png to avoid generating mermaid images via API
     "draw_mermaid_png",
+    # skip if it uses requests to load a file
+    "requests.get",
```
Comment (vbarda) [inline docs/_scripts/prepare_notebooks_for_ci.py:20]: @isahers1 this is very broad -- can we narrow it down?

File: docs/_scripts/prepare_notebooks_for_ci.py [hunk 2]
```diff
@@ -125,6 +127,11 @@ def add_vcr_to_notebook(
         "import msgpack",
         "import base64",
         "import zlib",
+        "import os",
+        "try:",
+        "    os.environ.pop(\"LANGCHAIN_TRACING_V2\")",
```
Comment (vbarda) [inline docs/_scripts/prepare_notebooks_for_ci.py:132]: you can just do `.pop(key, None)`, no need for try/except

Comment (vbarda) [reply docs/_scripts/prepare_notebooks_for_ci.py:132]: or just set to `"false"`

File: docs/_scripts/prepare_notebooks_for_ci.py [hunk 3]
```diff
@@ -70,10 +69,13 @@ def is_comment(code: str) -> bool:
...
 
-def has_blocklisted_command(code: str) -> bool:
+def has_blocklisted_command(code: str, metadata: dict) -> bool:
+    if 'hide_from_vcr' in metadata:
+        return True
+    
     code = code.strip()
-    for blocklisted_command in BLOCKLIST_COMMANDS:
-        if blocklisted_command in code:
+    for blocklisted_pattern in BLOCKLIST_COMMANDS:
```
Comment (vbarda) [inline docs/_scripts/prepare_notebooks_for_ci.py:77]: if you have time, would be great to change the other blocklisted cells to use this, but can be a separate PR

File: docs/_scripts/prepare_notebooks_for_ci.py [hunk 4]
```diff
@@ -108,7 +110,7 @@ def add_vcr_to_notebook(
         if all(is_comment(line) or not line.strip() for line in lines):
             continue
 
-        if has_blocklisted_command(cell.source):
+        if has_blocklisted_command(cell.source,cell.metadata):
```
Comment (vbarda) [inline docs/_scripts/prepare_notebooks_for_ci.py:113]: missing whitespace

**Review by vbarda — APPROVED** [review]

File: docs/_scripts/prepare_notebooks_for_ci.py (hunk 3 above)
Comment (isahers1) [reply docs/_scripts/prepare_notebooks_for_ci.py:77]: gonna merge this PR because there are a lot of cells to do this for and idk how to do this smartly (i.e. not manually)
